draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2267,20 +2487,20 @@ might be a 100Gb/s trunk port shared by ten 10GE client ports.
 
 The controller, when asked to set a 10GE client connectivity service
 between two muxponder's client ports, needs first to check whether
-there is already an existing WDM tunnel between the two muxponders
+there is already an existing DWDM tunnel between the two muxponders
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
there is already an existing WDM tunnel between the two muxponders
```